assets/dev/scss/editor/panel/panel.scss

```diff
@@ -114,11 +120,44 @@
 	}
 }
 
+#elementor-panel-get-pro-elements-sticky {
+	position: sticky;
+	bottom: 0;
+	font-size: 14px;
+	font-weight: 400;
+	line-height: 1.6;
+	margin-block-end: -10px;
+
+
+	.elementor-get-pro-sticky-message {
+		margin-top: -14px;
+		min-height: 40px;
+		display: block;
+		padding: 9px min(35px, 5%);
+		gap: 10px;
+		background-color: var(--e-a-color-white);
+		color: var(--e-a-color-black);
+		text-align: center;
+		box-shadow: 0px -10px 14px -6px rgba(0, 0, 0, 0.05);
```

Comment:
`box-shadow: 0px -10px 14px -6px rgba(0, 0, 0, 0.05);`

Does this work well in RTL?

Reply:
@louiswol94 I have tested it visually with RTL. I think it looks fine.